spec/android_update_release_notes_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+describe Fastlane::Actions::AndroidUpdateReleaseNotesAction do
+  after do
+    ENV['PROJECT_ROOT_FOLDER'] = nil
+  end
```

Comment:
What do you think to move this into an `ensure` block in each of the `in_tmp_dir` calls, instead of using an `after` block?

That being said, and given the actions are supposed to handle cases where `PROJECT_ROOT_FOLDER` is not set, and that the `release_notes_txt` or `changelog_md` path you pass to the action [here](https://github.com/wordpress-mobile/release-toolkit/pull/396/files#diff-b213d2371b4a182b49d9e5ae7f2b8794b1076ac56e15f0204bd4b49a12493fd0R61) are an absolute paths (which already includes the whole `tmp_dir` path in it)… I think your tests don't need to define a value for `ENV['PROJECT_ROOT_FOLDER']` to pass in the first place anymore… So you might as well remove all the lines in your tests that sets that env var and that should still pass?

Reply:
You're right! Removed in 091de20cabd5debfcc20a0a8df48f16e044b925b